README.md

```diff
@@ -275,3 +278,9 @@ Pop CLI would not be possible without these awesome crates!
 The entire code within this repository is licensed under the [GPLv3](LICENSE).
 
 Please [contact us](https://r0gue.io/contact) if you have questions about the licensing of our products.
+
+## Telemetry
```

Comment:
I think this should be moved to a subsection of the Installation section at the top. Users should be made aware of this at time of installation, rather than it be buried at the bottom of a long document. I know the latter wasnt intentional, but that is how it currently reads.